Functions/CatalogItems/Get-RsFolderContent.ps1

```diff
@@ -1,85 +1,82 @@
-﻿# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
+# Copyright (c) 2016 Microsoft Corporation. All Rights Reserved.
 # Licensed under the MIT License (MIT)
 
 function Get-RsFolderContent
 {
-	<#
-	.SYNOPSIS
-		List all catalog items under a given path.
-
-	.DESCRIPTION
-		List all catalog items under a given path.
-
-	.PARAMETER ReportServerUri
-		Specify the Report Server URL to your SQL Server Reporting Services Instance.
-		Has to be provided if proxy is not provided.
-
-	.PARAMETER ReportServerCredentials
-		Specify the credentials to use when connecting to your SQL Server Reporting Services Instance.
-
-	.PARAMETER Proxy
-		Report server proxy to use. 
-		Has to be provided if ReportServerUri is not provided.
-
-	.PARAMETER RsFolder
-		Path to folder on SSRS instance.
-
-	.PARAMETER Recurse
-		Recursively list subfolders with content.
-
-
-	.EXAMPLE
```

Comment:
@FriedrichWeinmann It looks like your version doesn't have the updated & additional Help example I put together,  Could you add them?